test/Syntax/tokens_invalids.swift

```diff
@@ -0,0 +1,79 @@
+// RUN: cat %s > %t
+
+// 5a is Z. "ZN" style marker is used for marker. N is number.
+
+// C2 is utf8 2 byte character start byte.
+// RUN: cat %t | sed 's/'$(echo -ne "\x5a1")'/'$(echo -ne "\xc2")'/g' > %t.sed
```

Comment:
```swift
// RUN: cat %t | sed 's/Z1/'$(echo -ne "\xc2")'/g' > %t.sed
                        ^^
```
doesn't work?

Reply:
No.  If I do so, `Z1` in sed line also replaced by this sed itself.
It makes problematic input source.

If Lexer is correct, it also skipped as line comment of course.
But for example it introduce other complex discussion about diagnostics about invalid character in comment.